Alpaca.Markets/RestClient.Polygon.cs

```diff
@@ -160,6 +160,40 @@ public Task<IEnumerable<IExchange>> ListExchangesAsync()
                 _polygonHttpClient, FakeThrottler.Instance, builder);
         }
 
+        /// <summary>
+        /// Gets list of historical minute bars for single asset from Polygon REST API endpoint.
+        /// </summary>
+        /// <param name="symbol">>Asset name for data retrieval.</param>
+        /// <param name="timespan">>Base length of bars. ("minute" or "day".)</param>
+        /// <param name="multiplier">>Number of bars to combine in each result.</param>
+        /// <param name="dateFromInclusive">Start time for filtering (inclusive).</param>
+        /// <param name="dateToInclusive">End time for filtering (inclusive).</param>
+        /// <param name="unadjusted">Set to true if the results should not be adjusted for splits.</param>
+        /// <returns>Read-only list of minute bars for specified asset.</returns>
+        public Task<IHistoricalItemsV2<IAgg>> ListAggregatesAsyncV2(
+            String symbol,
+            String timespan,
+            Int32 multiplier,
```

Comment:
This value can be negative? Any limitations which we can check on client side?

Reply:
I'm more inclined to leave it up to Polygon on how to handle the negatives or other nonsense cases, since that's something that they may want to change. (Contrived example, but maybe sending a negative multiplier causes the results to be returned in reverse order.) I think just making sure it's an Int is enough.